price-estimator/docker/Dockerfile

```diff
@@ -0,0 +1,24 @@
+# Dockerfile
+FROM rust:1-stretch as builder
+# Choose a workdir
+WORKDIR /usr/src/app
+# Copy sources
+COPY Cargo.toml .
+COPY Cargo.lock .
+COPY core core
+COPY driver driver
+COPY dex-contracts dex-contracts
+COPY e2e e2e
+COPY pricegraph pricegraph
+COPY price-estimator price-estimator
```

Comment:
Instead of copying all files 1 by 1, would it make sense to create a Dockerignore file for the folders we don't care about and then just copy `.` 

This would have the advantage that we don't have to add files here (and potentially  in other Dockerfiles) if we do a rename or add a new subproject.

Reply:
We already have a dockerignore file but I'm not sure if the same file is going to be correct for the driver and the price estimator. We should look into this but not sure I'll do it in this PR.